packages/contracts/test/token/erc20/governance-wrapped-erc20.ts

```diff
@@ -61,35 +64,42 @@ describe('GovernanceWrappedERC20', function () {
 
   beforeEach(async function () {
     defaultExistingERC20InitData = [existingErc20Name, existingErc20Symbol, 0];
-    erc20 = await TestERC20.deploy(...defaultExistingERC20InitData);
+    erc20 = await hre.wrapper.deploy('TestERC20', {
+      args: defaultExistingERC20InitData,
+    });
+
+    dao = await deployNewDAO(signers[0]);
 
-    const promises = defaultBalances.map(balance =>
-      erc20.setBalance(balance.account, balance.amount)
-    );
-    await Promise.all(promises);
+    for (let i = 0; i < defaultBalances.length; i++) {
+      await erc20.setBalance(
+        defaultBalances[i].account,
+        defaultBalances[i].amount
+      );
+    }
```

Comment:
Let's use `Promise.all` here to have at least a little bit speedier tests.

Reply:
I know, but Promise.all doesn't work for zksync :)